src/odemis/driver/piezomotor.py

```diff
@@ -470,15 +509,16 @@ def getStatus(self, axis):
         """
         return [int(i) for i in self._sendCommand(b'X%dU0' % axis)]
 
-    def isMovingClosedLoop(self, axis, target):
+    def isMovingClosedLoop(self, axis):
         """
         :param axis: (int) axis number
         :param target:  (float) target position for axes
-        :returns (bool): True if moving, False otherwise
+        :returns: (bool) True if moving, False otherwise
         """
-        axname = [name for name, num in self._axis_map.items() if num == axis][0]  # get axis name from number
-        position = float(self._sendCommand(b'X%dE' % axis)) / self._counts_per_meter[axname]
-        if abs(position - target) < 2 / self._counts_per_meter[axname]:  # 1 encoder count precision (difference less than 2 counts)
+        # Check d3 (third status value) bit 4 (targetLimit) and bit 1 (targetReached)
```

Comment:
It's bit *values* 4 (=2²) & 1(=2⁰). The bit number are 2 and 0. Please update this comment.
In practice, that means you could even write `if d3 & 5`, or more clearly `if d3 & 0b0101`.